view/omeka/site/item/show.phtml

```diff
@@ -18,6 +19,17 @@ $itemMedia = $item->media();
     ?>
     </div>
 <?php endif; ?>
+<h2 class="metadata-header">Descriptive Information</h2>
+<div class="property">
+    <h4><?php echo $translate('Permalink'); ?></h4>
+    <div class="values">
+        <div class="value">
+            <?php echo $this->hyperlink('https://' .
+                $_SERVER['HTTP_HOST'] .
```

Comment:
It isn't a huge deal but it would be best if could use the api here to generate the uri. The only edge case problem I can see is for use in non-secure, http configurations, but there may be other issues that I'm not thinking of.

Reply:
I believe [this](https://github.com/omeka/omeka-s/blob/d87e21c5da15e5a56ba7a6b3d17fa8c9807e2766/application/src/Api/Representation/ItemRepresentation.php#L86) is the part of the api that would get the item page in the context of a specific site. But it seems like the regular `url()` [function](https://github.com/omeka/omeka-s/blob/d87e21c5da15e5a56ba7a6b3d17fa8c9807e2766/application/src/Api/Representation/AbstractResourceRepresentation.php#L173) should detect if it is in the context of a site as well.